src/components/MoveAdder.test.ts

```diff
@@ -0,0 +1,84 @@
+import { IMove } from "battlemovr";
+import { expect } from "chai";
+import { stubBlankGame } from "../fakes.test";
+import { IPokemon } from "./Battles";
+import { MoveAdder } from "./MoveAdder";
+
+describe("MoveAdder", () => {
+    it("test Move Adder API", (): void => {
+        //Arrange
+        const { fsp } = stubBlankGame();
+        const pokemon: IPokemon = fsp.equations.newPokemon({
+            level: 5,
+            title: "SQUIRTLE".split(""),
+        });
+        const adder = new MoveAdder(fsp);
+        const peck: IMove = {
+            title: "Peck",
+            remaining: 10,
+            uses: 10,
+        };
+
+        //Act
+        adder.addMove(pokemon, peck, 1);
+
+        //Assert
+        expect(pokemon.moves[1].title === peck.title);
+    });
+
+    it("test Move Adder API - edge cases", (): void => {
```

Comment:
Continuing per the above comment - each unit test should test a single thing. That helps a lot when they start to fail, because you can see exactly what is or isn't working.